### DISCUSSION TITLE
Why this exists!

### DISCUSSION THREAD

**Discussion by martincollignon** [Klimabevaegelsen/landbruget.dk#20] [Announcements]
Halløj! 🤓

I'm Martin Collignon ([Linkedin](https://www.linkedin.com/in/martincollignon/), [Twitter](https://twitter.com/martincollignon), [Bluesky](https://bsky.app/profile/martincollignon.bsky.social))! I've been fighting for a carbon tax on agriculture in Denmark, and before that for a carbon tax on the industry. Semi-succesfully, since Denmark will getting both, albeit not at cost that is high enough at all.

When I debate, I do my best to get my facts straight. And the agricultural sector is a bonanza of facts, because they have to report a lot of things to get the juicy subsidies. Many of the facts I found came through sporadic newspaper articles, and often at either very high level, or ultra-specific about one farm or one person. I've seen first hand how long time it takes to gather data and to wait for responses from government about data access requests.

This is my attempt - that hopefully will become our attempt - at gathering all facts one place, so it's easily accessible by anyone. 

So everyone can get the facts straight.

Feel free to help 🙏

**Comment by iamfrank** [comment]
Interesting project. Are there plans for hosting the app on a website for public view?

Best of luck.

**Reply by martincollignon** [reply]
> Interesting project. Are there plans for hosting the app on a website for public view?
> 
> Best of luck.

100%. I've just focused on getting and transforming the data so far (soon there!), and that is obvious next step.

 Would love help to think about how to best visualise this, as I have poor taste and I'm too much in the material :)

**Reply by iamfrank** [reply]
Might be able to extend a hand in that field. I'll fork the repo and take a look post Christmas.
